pkgs/development/libraries/libdeltachat/default.nix

```diff
@@ -31,13 +31,13 @@ let
   };
 in stdenv.mkDerivation rec {
   pname = "libdeltachat";
-  version = "1.140.2";
+  version = "1.142.1";
```

Comment:
1.142.3 is out.

Reply:
But we want to ship the exact version Delta chart desktop requires.